pkg/metrics/metrics.go

```diff
@@ -0,0 +1,59 @@
+package metrics
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"sync"
+
+	"github.com/golang/glog"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+const namespace = "kube_parrot"
+
+var (
+	BgpAddNeighborSuccess = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: namespace,
+			Name:      "add_bgp_neighbors_success",
+			Help:      "Counter for successful neighbor add operations.",
+		},
+		[]string{"node"},
+	)
+
+	BgpAddNeighborFailure = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: namespace,
+			Name:      "add_bgp_neighbors_failure",
+			Help:      "Counter for failed neighbor add operations.",
+		},
+		[]string{"node"},
+	)
+)
+
+func init() {
+	prometheus.MustRegister(
+		BgpAddNeighborSuccess,
+		BgpAddNeighborFailure,
+	)
+}
```

Comment:
not sure but potentially we could remove this as neighbors not in `established` state will be catched by `GetNeighborBgpState` anyway